host/tests/integration_tests/evil/fs.rs

```diff
@@ -26,6 +26,8 @@ const PATHS: &[&str] = &[
     "/proc/self",
     "/sys",
     "/tmp",
+    "\0",
+    "/x/..",
 ];
```

Comment:
I notice this list doesn't contain `/bin`, `/boot`, `/lib` . We (I) should (can) write a ticket for further hardening that describes threat vectors based around those folders (for example, linking an evil lib) (?)